apps/emqx/src/emqx_config.erl

```diff
@@ -90,6 +92,13 @@
 -define(ZONE_CONF_PATH(ZONE, PATH), [zones, ZONE | PATH]).
 -define(LISTENER_CONF_PATH(TYPE, LISTENER, PATH), [listeners, TYPE, LISTENER | PATH]).
 
+-define(UNSAFE_ATOM_CONF_PATH(PATH, EXP, EXP_ON_FAIL),
```

Comment:
Macros must be wrapped in `((fun() -> ... end)())` to avoid variable bindings escaping the scope. 
Why is it a macro in the first place? It's only used once, as far as I see.

Reply:
I tried to be consistent with `ATOM_CONF_PATH` macro

Reply:
I would prefer to keep the code as simple as possible.

Reply:
Yeah, I think we need to raise the question why configuration path is a list of atoms.

But for now, I will implement a workaround as a function